cancer_ai/utils/config.py

```diff
@@ -265,14 +265,14 @@ def add_validator_args(cls, parser):
     )
 
     parser.add_argument(
-        "--wandb.project_name",
+        "--wandb_project_name",
         type=str,
         help="The name of the project where you are sending the new run.",
         default="template-validators",
     )
 
     parser.add_argument(
-        "--wandb.entity",
+        "--wandb_entity",
```

Comment:
are we changing standard of arguments naming to straight argument name like here instead of category first approach?
ex.
now its --wandb_entity
previously it was --wandb.entity
if yes can we look over all other arguments and make this approach consistent? Some of them are still using categorized approach like
models.model_dir